stdlib/public/core/Optional.swift

```diff
@@ -182,15 +182,15 @@ public enum Optional<Wrapped> : ExpressibleByNilLiteral {
   ///         return overflowed ? nil : result
   ///     }
   ///     print(nonOverflowingSquare)
-  ///     // Prints "Optional(1746)"
+  ///     // Prints "Optional(1764)"
   ///
   /// - Parameter transform: A closure that takes the unwrapped value
   ///   of the instance.  
   /// - Returns: The result of the given closure. If this instance is `nil`,
   ///   returns `nil`.
   @_inlineable
   public func flatMap<U>(
-    _ transform: (Wrapped) throws -> U?
+    _ transform: (Unwrapped) throws -> U?
```

Comment:
sorry, my English is pool, and can I think 'Wrapped' is the value  unwrapped from a wrapped value ?